### DISCUSSION TITLE
The bots do not take off the oxygen mask when they put on the diving suit.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14091] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The bots do not take off the oxygen mask when they put on the diving suit. 
If the tank in the suit runs out, the bot does not change the tank in the suit, but puts on the mask.
The mask spends air 2 times faster!

### Reproduction steps

1. give the bot an oxygen mask.
2. order bot to put on a diving suit
3. remove the balloon from the diving suit the bot is wearing
4. the bot will put on a mask and will walk around in a mask and a diving suit without a tank.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Thanks for the report. Seems to only happen if you wear a diving suit in a room where diving mask is needed, and have a mask in the inventory. For example a room lacking with oxygen or flooding without being pressurized. Created a ticket: #14094.

**Comment by itchyOwl** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5414.
